### PR TITLE
Article about the importance of architecture

### PR DESCRIPTION
**Pull request by romanutti**

### CODE REVIEW

File: importance-of-architecture.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+However, this ignores that developers are their software's stakeholders too. Software contains the term *soft*. It is intended to be soft: Changing software and their behavior should be easy - otherwise, we would have called it *hard*ware. When the business changes a requirement or aims for a new feature, that change should be easily accomplishable. We use **architecture** to structure our applications to minimize the efforts required to build and maintain a system. Because architectural aspects often require a technical understanding of the system, it is the task of the developer (and architect) to ensure architecture in a software project.
+
+Both, implementing the behavior of the software and enforcing its architectural aspects require resources. As time and money are limited in software projects (at least those of which I have been a part), we ask ourselves which one to prioritize.
+
+# The Mere-Urgency Effect a.k.a. Why We're Bad at Prioritization
```
Comment (linkdotnet) [inline importance-of-architecture.md:19]: Small SEO hint: h1 should reflect the title.

https://developers.google.com/style/headings#formatting-a-heading-or-title

File: importance-of-architecture.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+Already President Dwight D. Eisenhower recognized that we have difficulties answering that question. At the 1961 Century Association, he stated:
+
+> Who can define for us with accuracy the difference between the long and short term! Especially whenever our affairs seem to be in crisis, we are almost compelled to give our first attention to the urgent present rather than to the important future.
+
+This observation was recently confirmed by a [study](https://www.researchgate.net/publication/327103488_The_Mere_Urgency_Effect) in the Journal of Consumer Research. Researchers examined how people decide what to work on when faced with tasks of mixed urgency and importance. Across five experiments, they observed the following pattern:
+We tend to favor time-sensitive tasks over tasks that are less urgent, regardless of their long-term payoffs. This psychological quirk is called the *Mere-Urgency Effect.*
+
+# The Long-Term View
```
Comment (linkdotnet) [inline importance-of-architecture.md:28]: I'd propose to make `h2` elements out of it

Comment (romanutti) [reply importance-of-architecture.md]: Fixed, thanks for pointing it out!

**Review by r3dDoX — APPROVED** [review]
Looks good to me

**Pull request merged**